src/js/common/schemaform/actions.js

```diff
@@ -1,5 +1,5 @@
 import Raven from 'raven-js';
-import _ from 'lodash/fp';
+import _ from '../utils/lodash-replacements';
```

Comment:
I kind of think of these as data utils, but maybe that's too generic?

Reply:
I wasn't sure where to put it them or how we should import them (as `_`? in `{ get, set }`? from different files entirely?) What's your opinion?

Reply:
Ohh, unless your comment is directed at the name `lodash-replacements`. Hmm?

Reply:
Yeah, I meant `lodash-replacements`.